.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce**
+If possible, provide a recipe for reproducing the error.
+
+**What did you expect to see?**
+A clear and concise description of what you expected to see.
+
+**What did you see instead?**
+A clear and concise description of what you saw instead.
+
+**What version and what artifacts are you using?**
+Version: (e.g., `v0.4.0`, `1eb551b`, etc)
+Artifacts: (e.g., `api`, `sdk`, etc)
```

Comment:
```suggestion
Artifacts: (e.g., `api`, `sdk`, etc)
JDK: (e.g., "AdoptOpenJDK 11.0.6")
JRE: (e.g., "Oracle JRE 8u251")
OS: (e.g., "Ubuntu 20.04")
```

Reply:
can you add this quickly as a PR, @arminru ?

Reply:
sure, @jkwatson!